client/rpc-core/txpool/Cargo.toml

```diff
@@ -8,13 +8,12 @@ license = 'GPL-3.0-only'
 repository = 'https://github.com/PureStake/moonbeam/'
 
 [dependencies]
-# ethereum = { git = "https://github.com/rust-blockchain/ethereum", branch = "master", features = ["with-codec"] }
-ethereum = { git = "https://github.com/notlesh/ethereum", branch = "notlesh-moonbeam-v0.6", features = ["with-codec"] }
+ethereum = { git = "https://github.com/rust-blockchain/ethereum", branch = "master", default-features = false, features = ["with-codec"] }
```

Comment:
Personally, I support leaving the commented one. It's an extra reminder that we're using a non-default repo.